### PR TITLE
ETL Day3 Assignment Submission

### PR DESCRIPTION
**Pull request by Saphall**

### CODE REVIEW

**Comment by Saphall** [comment]
confused where to open pull request: `main` or in `branch`

File: Week3/Day2/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## ETL Design Assignment 2.
```
Comment (ShishirBhandari) [inline Week3/Day2/README.md]:
```suggestion
## ETL Design Assignment 3.
```

File: Week3/Day2/docs/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+```  
+This is SQL file to create `raw_timesheet_archieve` table to archieve `raw_timesheet` table after extraction.
+
+
+All entities are again declared VARCHAR for same problem above.
+
+
+## 2. `database_connection.py` file in  src/pipeline/ :
+
+We connect to Postgresql database regularly in each script through `psycohpg2` and use the cursor with connection.
```
Comment (ShishirBhandari) [inline Week3/Day2/docs/Documentation.md:103]: typo mistakes in many places

File: Week3/Day2/schema/create_employee_table_archieve.sql [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    employee_role VARCHAR(500),
+    salary VARCHAR(500),
+    hire_date VARCHAR(500),
+    terminated_date VARCHAR(500),
+    terminated_reason VARCHAR(500),
+    dob VARCHAR(500),
+    fte VARCHAR(500),
+    location VARCHAR(500),
+    file_name VARCHAR(500)
+);
```
Comment (ShishirBhandari) [inline Week3/Day2/schema/create_employee_table_archieve.sql:17]: Where is the primary key ?

File: Week3/Day2/src/pipeline/extract_employee_data_from_json.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import json
+from database_connection import *;
+from psycopg2.extras import Json
+from archieveTable import archieveTable
+
+def extract_employee_data_from_json(filePath):
```
Comment (ShishirBhandari) [inline Week3/Day2/src/pipeline/extract_employee_data_from_json.py:6]: Try to break down these large functions into smaller ones.

File: Week3/Day2/README.md (hunk 1 above)
Comment (Saphall) [reply Week3/Day2/README.md]: Dai actually this is `Day2/Readme.md` and correct. I updated mistakes in `Day3/Readme.md`. [HERE](https://github.com/Saphall/Leapfrog_Data-Engineering_Assignments/commit/7b0310ff554d473aac500ddfc8f32b72bde364c8#diff-9060e8fa0c9c0320990a5cd88852009d878e64cd91917bf706f6550241a011f6) already.
This is  PR for `Day 3` branch . [HERE](https://github.com/Saphall/Leapfrog_Data-Engineering_Assignments/tree/Day3_Assignment/Week3/Day3).  

Actually the branch creation was problem and I will resolve the conflict.

File: Week3/Day2/docs/Documentation.md (hunk 2 above)
Comment (Saphall) [reply Week3/Day2/docs/Documentation.md:103]: Dai I corrected major typos in `Day2` branch for this file but was not reflected in this `Day 3` branch. (branch creation problem). I will resolve it while merging.

File: Week3/Day2/schema/create_employee_table_archieve.sql (hunk 3 above)
Comment (Saphall) [reply Week3/Day2/schema/create_employee_table_archieve.sql:17]: Dai this is raw table so.

File: Week3/Day2/src/pipeline/extract_employee_data_from_json.py (hunk 4 above)
Comment (Saphall) [reply Week3/Day2/src/pipeline/extract_employee_data_from_json.py:6]: ok dai.

**Pull request merged**